src/components/modals/SurveyModal.tsx

```diff
@@ -6,47 +6,77 @@ import Animated, { Easing, useAnimatedStyle, useSharedValue, withTiming } from '
 import { sprintf } from 'sprintf-js'
 
 import { useHandler } from '../../hooks/useHandler'
+import { getLocaleOrDefaultString } from '../../locales/intl'
 import { lstrings } from '../../locales/strings'
 import { config } from '../../theme/appConfig'
 import { useDispatch } from '../../types/reactRedux'
-import { infoServerData } from '../../util/network'
 import { logEvent } from '../../util/tracking'
 import { shuffleArray } from '../../util/utils'
 import { ModalButtons } from '../buttons/ModalButtons'
 import { EdgeCard } from '../cards/EdgeCard'
+import { Airship } from '../services/AirshipInstance'
 import { cacheStyles, Theme, useTheme } from '../services/ThemeContext'
 import { EdgeText, HeaderText, Paragraph, SmallText } from '../themed/EdgeText'
 import { SimpleTextInput } from '../themed/SimpleTextInput'
 import { Radio } from '../themed/ThemedButtons'
 import { EdgeModal } from './EdgeModal'
 
-const SURVEY_OPTS = [
-  { label: lstrings.survey_opt_youtube, selected: false },
-  { label: lstrings.survey_opt_search_engine, selected: false },
-  { label: lstrings.survey_opt_x_twitter, selected: false },
-  { label: lstrings.survey_opt_in_person_event, selected: false },
-  { label: lstrings.survey_opt_personal_referral, selected: false },
-  { label: lstrings.survey_opt_article, selected: false }
+interface SurveyCategory {
+  catKey: string
+  label: string
+  selected: boolean
```

Comment:
I'm not going to be too picky, since this is temporary anyhow, but this `selected` flag should not exist. Everything should use the `selectedIndex` instead, which is actual component state.

However, no need to change anything now. Just make sure the later rewrite includes this fix.